src/main/resources/org/jenkinsci/plugins/githubissues/GithubIssueNotifier/config.jelly

```diff
@@ -0,0 +1,25 @@
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form">
+    <f:block>
+        <f:optionalBlock name="useCustomTemplate" inline="true" title="Customize issue template" checked="${instance.useCustomTemplate}">
+            <f:entry title="Issue Title Template" field="issueTitle">
+                <f:textbox />
+            </f:entry>
+            <f:entry title="Issue Body Template" field="issueBody">
+                <f:textarea />
+            </f:entry>
+            <f:entry title="Issue Label" field="issueLabel">
+                <f:textbox />
+            </f:entry>
+            <f:entry title="Reopen issue" field="issueReopen">
+                 <f:checkbox default="true"/>
```

Comment:
I'll probably default this to `false` - Reopening a previous issue isn't very useful in a lot of cases, particularly if the existing issue is very old. I guess a setting like "Reopen the issue if it's less than X days old, otherwise create a new one" would be helpful.